internal/services/servicebus/servicebus_subscription_resource_test.go

```diff
@@ -369,3 +384,41 @@ func (ServiceBusSubscriptionResource) updateDeadLetteringOnFilterEvaluationExcep
 	return fmt.Sprintf(testAccServiceBusSubscription_tfTemplate, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger,
 		"dead_lettering_on_filter_evaluation_error = false\n")
 }
+
+func (ServiceBusSubscriptionResource) clientAffineSubscriptionEnabled(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%[1]d"
+  location = "%[2]s"
+}
+
+resource "azurerm_servicebus_namespace" "test" {
+  name                = "acctestsbn-%[1]d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku                 = "Premium"
+  capacity            = 1
+}
+
+resource "azurerm_servicebus_topic" "test" {
+  name         = "acctestservicebustopic-%[1]d"
+  namespace_id = azurerm_servicebus_namespace.test.id
+}
+
+resource "azurerm_servicebus_subscription" "test" {
+  name                                 = "_acctestsub-%[1]d_"
+  topic_id                             = azurerm_servicebus_topic.test.id
+  max_delivery_count                   = 10
+  auto_delete_on_idle                  = "PT5M"
+  lock_duration                        = "PT1M"
+  dead_lettering_on_message_expiration = true
+	%[3]s
+  client_scoped_subscription_enabled       = true
```

Comment:
could we clean this up? the %[3]s is confusing

Reply:
will do